fusor-ember-cli/app/routes/deployment.js

```diff
@@ -133,20 +133,15 @@ export default Ember.Route.extend(DeploymentRouteMixin, UsesOseDefaults, {
           "Authorization": "Basic " + self.get('session.basicAuthToken')
         }
       }).then(
-        function (response) {
-          var uuid = response.id;
-          console.log('task uuid is ' + uuid);
-          deployment.set('foreman_task_uuid', uuid);
-          deployment.save().then(
-            function () {
-              controller.set('showSpinner', false);
-              return self.transitionTo('review.progress.overview');
-            },
-            function () {
-              controller.set('showSpinner', false);
-              controller.set('errorMsg', 'Error in saving UUID of deployment task.');
-              controller.set('showErrorMessage', true);
-            });
+        function (_) {
```

Comment:
I don't think we've done it yet, so it's not a big deal, but I was refering to pushing the returned object directly into the store instead of making a second async request.

https://guides.emberjs.com/v1.13.0/models/pushing-records-into-the-store/